LSDtest1/views.py

```diff
@@ -79,7 +79,7 @@ def run_single_display(request):
             
             # Run other commands
             execute_command = (
-                "sudo -S bash -c '"
+                "sudo su '"
                 "python3 -m venv /env1; "
                 "source /env1/bin/activate; "
                 "/home/jordan/env1/pyscripts/strandtest.py'"
```

Comment:
Make sure you close the shell process to help eliminate timeout errors / Bugs!

Reply:
![image](https://github.com/WSU-4110/Live-Sports-Display/assets/114885319/b161f005-6147-4e7d-839b-acad5dcda1e8)

Reply:
thank you, should I setup my environment from boot? how are you closing your connection?

Reply:
you should set your environment from boot using crontab -e, which registers your .py file and boots at startup.... an example of this would be. 

@reboot python3 /home/timkosinski/pi-power-button/listen-for-shutdown.py

referring to closing the session, all I have is that my SSH function only opens up the file and closes it... anytime that file is updated, the program runs again...

![image](https://github.com/WSU-4110/Live-Sports-Display/assets/127711040/260eae1b-b2e9-4ad8-b873-422b557aa55a)

Hope this helps!!!

if you have any questions let me know!

Reply:
sounds good, ill make the changes on my pi.